### DISCUSSION TITLE
Onchange not emit after setValue

### DISCUSSION THREAD

**Discussion by danbn-100295** [react-hook-form/react-hook-form#2762] [Q&A]
Here is my code:
![Annotation 2020-08-29 144126](https://user-images.githubusercontent.com/61224981/91631707-c5a6cb00-ea05-11ea-9b27-8419a092ae5c.png)
I expect that every time i call clearSaerch , onChange in render props emit but it seem to be it not emit and only update value on TextField
Please help me !

**Comment by bluebill1049** [comment]
codeSandbox?
